plugins/contentful.js

```diff
@@ -0,0 +1,19 @@
+// ./plugins/contentful.js
+
+require('dotenv').config();
```

Comment:
Let's stick this in server/index.js

Reply:
It doesn't seem to be required at all. However removing it resulted in some console errors, so I've simplified the export funcion of this file.